helmfile.d/traefik.yaml

```diff
@@ -0,0 +1,20 @@
+repositories:
+  - name: traefik
+    url: 
+releases:
+- name: traefik
+  chart: stable/traefik
+  namespace: kube-system
+  version: 1.3.0
+  wait: true
+  timeout: 300
+  atomic: true
+values:
+  - defaultBackend:
```

Comment:
Normally we use value files instead of defining them directly in the helmfile

Reply:
The only traefik helmchart I could find doesn't even mention default backend so I don't think this even does anything.

Reply:
https://github.com/traefik/traefik-helm-chart

Is there any reference I can pick values from. And should there be a different values file ?